src/Entity/Adherent.php

```diff
@@ -435,23 +452,17 @@ public function getLastLoggedAt(): ?\DateTime
         return $this->lastLoggedAt;
     }
 
-    /**
-     * @return array|null
-     */
-    public function getInterests(): ?array
+    public function getInterests(): array
```

Comment:
Yeah as long as the choice field is set as multiple an array will be used as empty value (which must be since our model **is** an array).